keyboards/protokeeb/rev1/mcuconf.h

```diff
@@ -0,0 +1,36 @@
+/* Protokeeb Copyright 2024 A-Tech Officials (@atechofficials)
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU General Public License for more details.
+ * 
+ * You should have received a copy of the GNU General Public License
+ * along with this program. 
+ * If not, see <https://www.gnu.org/licenses/gpl-3.0.html>.
+ */
+
+#pragma once
+
+#include_next <mcuconf.h>
+
+/* RP2040 I2C Driver Selection */
+#undef RP_I2C_USE_I2C0
+#define RP_I2C_USE_I2C0 TRUE
+
+#undef RP_I2C_USE_I2C1
+#define RP_I2C_USE_I2C1 FALSE
+
+// #define RP_I2C_BUSY_TIMEOUT 50
+// #define RP_I2C_USE_DMA TRUE
+
+/* RP2040 SPI Driver Selection */
+#undef RP_SPI_USE_SPI0
+#define RP_SPI_USE_SPI0 TRUE
+
+#undef RP_SPI_USE_SPI1
+#define RP_SPI_USE_SPI1 FALSE
```

Comment:
What is SPI used for on this keyboard? If nothing, this should be removed.

Reply:
I was testing other functionalities of the QMK firmware. Removed SPI preprocessor directives.